charts/kube-bench-metrics/templates/daemonset.yaml

```diff
@@ -29,6 +29,10 @@ spec:
           - {{ .Values.versionOverride | quote}}
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           volumeMounts:
+            {{- if and ( not .Values.kubeadmVolumes.enabled ) ( not .Values.extraVolumeMounts ) }}
+            []
+            {{- end }}
```

Comment:
Is this really needed?
I think it will accept a empty `volumeMounts` but I might be wrong here?

Reply:
I assumed that it would be invalid yaml without it, but I just tried it with kubectl and it worked fine.
Removed it now.